src/mscorlib/shared/System/ReadOnlySpan.Fast.cs

```diff
@@ -149,6 +149,19 @@ internal ReadOnlySpan(ref T ptr, int length)
 #endif
         }
 
+        /// <summary>
+        /// Returns a reference to the 0th element of the Span. If the Span is empty, returns null reference.
+        /// It can be used for pinning and is required to support the use of span within a fixed statement.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ref readonly T GetPinnableReference()
+        {
+            if (_length == 0)
```

Comment:
It maybe better to write this as `return (_length != 0) ? ref _pointer.Value : ref Unsafe.AsRef<T>(null)`.